src/Microsoft.ServiceFabric.Actors.Tests/Generator/ActorNameFormatTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.ServiceFabric.Actors.Generator;
```

Comment:
Also please look at build.ps1 and add an option to build test code to hook these into building from outiside of VS. Consider dropping test assemblies into drop\tests folder. Easier to run all tests from this one location from command line.